scripts/compute_mappings/compute_ancestor_mapping.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+# coding: utf-8
+# flake8: noqa
+
+# # Ontology ancestors mapping
+# This notebook can be used to compute mappings between nodes in an ontology files and their ancestors.
+# It will:
+# 1. Load the required ontology files (human, mouse, UBERON)
+# 1. For `human` and `mouse`, start from each node available in the ontology
+# 1. for `UBERON`, only start from a defined, hardcoded subset of leaf nodes which is relevant for our datasets
+# 1. Recursively compute ancestors using the `is_a` relationship and `part_of` restriction (defined as `BFO_0000050`)
+# 1. Create mappings from each node to its ancestors
+# 1. Write a json file with these mappings (a dictionary of lists)
+
+# In[1]:
+
+
+from owlready2 import get_ontology  # noqa
```

Comment:
We should add unit test for this code now that it's not a note book.

Reply:
I'll make an issue for this separately